orchagent/aclorch.cpp

```diff
@@ -493,17 +491,112 @@ bool AclRuleL3::validateAddAction(string attr_name, string _attr_value)
     {
         value.aclaction.parameter.s32 = SAI_PACKET_ACTION_DROP;
     }
+    else if (attr_value.find(PACKET_ACTION_REDIRECT) != string::npos)
```

Comment:
I don't want to split the attr_value before the if, because we have ':' only for REDIRECT. And I don't know what else we might introduce as a PACKET ACTION here. So I made all changes as local as possible.